config/custom_roundstart_items.txt

```diff
@@ -5,4 +5,4 @@
 //Recommend defining one job per line, but do what you want.
 //test1|Secondary Memenamefortesting|testjob1/test job 2/ALL|/obj/item/gun/energy/laser=3;/obj/item/gun/energy/laser/retro=1;/obj/item/gun/energy=2
 //test1|Memename Lastname|testjob1|/obj/item/device/aicard=3;/obj/item/device/flightpack=1;/obj/item/gun/energy=3
-JayEhh|ALL|ALL|/obj/item/custom/ceb_soap=1
\ No newline at end of file
+Galloping_Ghost|ALL|ALL|/obj/item/clothing/head/fingerhood=1
```

Comment:
We do not use the custom_roundstart_items.txt file to give donor items to the people they belong to. We have them integrated with the character loadout system. 

You've already added the item in donor.dm.